lib/ActionClient.js

```diff
@@ -31,28 +31,33 @@ class ActionClient extends EventEmitter {
     const nh = options.nh;
 
     // FIXME: support user options for these parameters
+    const goalOptions = Object.assign({ queueSize: 10, latching: true }, options.goal);
```

Comment:
More of a question - so it's expected that the goal and cancel pubs will have a default queueSize of 10 and the others stay at 1?